src/main/java/com/execution/Executor.java

```diff
@@ -0,0 +1,82 @@
+package com.execution;
+
+import com.util.DbUtil;
+import com.model.Account;
+import com.model.Profile;
+
+public class Executor {
+    private static final int ROW_NUMBER = 500;
+    private final Account accounts = new Account();
+    private final Profile profiles = new Profile();
+    private final DbUtil dbUtil = DbUtil.getINSTANCE();
+    private Account petroAccount;
+    private Profile petroProfile;
+    private Account vasilAccount;
+    private Profile vasilProfile;
+    private Profile profileFromTable;
+    private Account accountFromTable;
+
+    public void startProcessing() {
+        createPetroAccountAndProfile();
+        insertAccountAndProfileIntoTable(petroAccount, petroProfile);
+        printAccountsAndProfiles();
+
+        createVasilAccountAndProfile();
+        updateAccountAndProfileById(vasilAccount, vasilProfile);
+        printAccountsAndProfiles();
+
+        removeAccountAndProfile(vasilAccount, vasilProfile);
+        printAccountsAndProfiles();
+
+        getAccountAndProfileFromTables(ROW_NUMBER);
+        tryToPrintAccountAndProfileInformation(accountFromTable, profileFromTable);
+    }
+
+    private void tryToPrintAccountAndProfileInformation(Account account, Profile profile) {
```

Comment:
try to print? Why "try?"

Reply:
> try to print? Why "try?"

I wasn't sure how to name the method correctly when the result could be different from what you expect

fixed